manager/testdata/unittests/module-read-write.yaml

```diff
@@ -13,6 +13,9 @@ spec:
   capabilities:
     - capability: read
       scope: workload
+      actions:
+        - name: RedactAction
+        - name: RemoveAction
       api:
         connection:
           name: fybrik-arrow-flight
```

Comment:
fybrik-flight

Reply:
add dataFormat = arrow

Reply:
I believe "arrow" format has been deprecated as well as "table" for db2.